src/types.ts

```diff
@@ -1115,5 +1115,56 @@ export interface SearchReponse {
 }
 
 export interface TransactionResult {
-  txHash: string
-}
\ No newline at end of file
+  txHash: string;
+}
+
+export interface ChatConfigureParamsBase {
+  objectId: string;
+}
+
+export interface OpenAIChatConfigureParams extends ChatConfigureParamsBase {
+  provider: 'openai';
+  api: 'assistants';
+}
+
+export type ChatConfigureParams = OpenAIChatConfigureParams;
+
+export interface AssistantCreateParamsBase {
+  tokenId: string;
+  name: string;
+  instructions: string;
+  description?: string | null;
+}
+
+export interface OpenAIAssistantCreateParams
+  extends OpenAIChatConfigureParams,
+    AssistantCreateParamsBase {
+  /**
+   * Name of the model to use. You can see the
+   * [OpenAI model overview](https://platform.openai.com/docs/models/overview)
+   * for description of them.
+   */
+  model:
+    | 'gpt-4-1106-preview'
+    | 'gpt-4'
+    | 'gpt-4-32k'
+    | 'gpt-3.5-turbo-1106'
```

Comment:
(question) Is the assistant api supported with gpt-3.5 models?

Reply:
Yes! The `gpt-3.5-turbo` model is support `assistants` and `chat(completion)` api.

Reply:
Good to know that, thanks!